client/src/components/Translations.jsx

```diff
@@ -17,13 +17,20 @@ const useStyles = makeStyles(theme => ({
 export default function TranslationsMenu() {
   const classes = useStyles();
   const { i18n, t } = useTranslation();
+  const [loaded, setLoaded] = useState(false);
+
+  const setLanguage = () => {
+    setLoaded(true)
+    i18n.changeLanguage(window.navigator.language.slice(0, 2));
+  }
 
   const handleClick = event => {
     i18n.changeLanguage(event.target.value);
   };
 
   return (
     <div>
+      {loaded ? null : setLanguage()}
```

Comment:
Couldn't we just call `setLanguage()` in a `useEffect()` hook?

Reply:
Done